src/components/ImageCarousel.tsx

```diff
@@ -1,37 +1,51 @@
 import React, { ReactElement } from 'react'
+import { Keyboard, useWindowDimensions } from 'react-native'
 import ImageViewer from 'react-native-image-zoom-viewer'
 import { Pagination } from 'react-native-snap-carousel'
 import styled from 'styled-components/native'
 
 import { ImagesType } from '../constants/endpoints'
 import { COLORS } from '../constants/theme/colors'
 
-const ImageView = styled.View`
-  height: 35%;
+const ImageView = styled.View<{ height: number }>`
+  height: ${({ height }) => height}px;
 `
-const StyledImage = styled.Image`
-  height: 100%;
+const StyledImage = styled.Image<{ height: number; minimized: boolean }>`
+  height: ${({ height }) => height}px;
+  ${({ minimized }) =>
+    minimized &&
+    `
+    aspect-ratio: 1.5;
+    margin: 0 auto;
+  `}
 `
-const PaginationView = styled.View`
+const PaginationView = styled.View<{ topPosition: number }>`
   position: absolute;
   left: 0px;
   right: 0px;
-  top: 10px;
+  top: ${({ topPosition }) => topPosition}px;
 `
 
 interface ImageCarouselPropsType {
   images: ImagesType
+  minimized?: boolean
 }
+
 interface ItemType {
   source: {
     uri: string
   }
 }
+
 interface ImageUrlType {
   url: string
 }
 
-const ImageCarousel = ({ images }: ImageCarouselPropsType): ReactElement => {
+const ImageCarousel = ({ images, minimized = false }: ImageCarouselPropsType): ReactElement => {
+  const { height: deviceHeight } = useWindowDimensions()
+  const heightPercent = minimized ? 17.5 : 35
+  const viewerHeight = (deviceHeight * heightPercent) / 100
```

Comment:
Maybe a comment would make sense here. Also avoid magic numbers (perhaps better to write `17.5` as `35 / 2`)